benchmark/performance_utils.py

```diff
@@ -99,3 +99,10 @@ def ternary_args(dtype, batch, size):
     inp2 = torch.randn([batch, size], dtype=dtype, device="cuda")
     inp3 = torch.randn([batch, size], dtype=dtype, device="cuda")
     return inp1, inp2, inp3
+
+
+def where_args(dtype, batch, size):
```

Comment:
you can define where_args in test_perf_where. it's more clear for users.

Reply:
this file has conflict with other PR which separate the Op_args